### PR TITLE
Adding categories table

### PR DESCRIPTION
**Pull request by jgtoriginal**
* Wrote a python script to crawl over categories and build the categories table on the db.
* Categorized cards on db
* Fixed many typos on image names (db, files, tokens)
* Deleted redundant db.
* Deleted miss placed certificate file.
* Fixed `xcworkspace`

### CODE REVIEW

File: src/helpers/validators/validators.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-export const isValidPassword = item => {
-  return true;
-}
+export const isValidEmail = email => {
+    return email.match(/^\w+(\.\w+)*@[a-zA-Z_]+?\.[a-zA-Z]{2,3}(\.[a-zA-Z]{2}){0,1}$/);
```
Comment (Zurc) [inline src/helpers/validators/validators.js]: cuando vi este pattern me pareció interesante compartir un par de artículos relacionados... ;)

https://www.w3resource.com/javascript/form/email-validation.php
https://medium.com/hackernoon/the-100-correct-way-to-validate-email-addresses-7c4818f24643
https://www.regular-expressions.info/email.html

File: src/assets/pecsCards/index.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ export const Data = [
 	{ card_id:37, name: "Do_laundry", url: require("../pecsCards/Do_laundry.jpg"), name_it: "Fare il bucato" },
 	{ card_id:38, name: "Summer", url: require("../pecsCards/Summer.jpg"), name_it: "Estate" },
 	{ card_id:39, name: "Turtle", url: require("../pecsCards/Turtle.jpg"), name_it: "Tartaruga" },
-	{ card_id:40, name: "Raspberry", url: require("../pecsCards/Raspberry.jpg"), name_it: "Lampone" },
+	{ card_id:40, name: "Rasberry", url: require("../pecsCards/Rasberry.jpg"), name_it: "Lampone" },
```
Comment (Zurc) [inline src/assets/pecsCards/index.js]: Ehhh.... no me arruinés la fruta.... me pusiste la version cumbia vo'...
Los de Cambridge eligieron el nombre de la fruta (con la 'p') para sus mini-computadoras...
https://www.raspberrypi.org/forums/viewtopic.php?f=2&t=2657

File: src/assets/pecsCards/index.js [hunk 3]
```diff
@@ -261,11 +261,11 @@ export const Data = [
 	{ card_id:260, name: "Bad_Smell", url: require("../pecsCards/Bad_Smell.jpg"), name_it: "Puzza" },
 	{ card_id:261, name: "Pick", url: require("../pecsCards/Pick.jpg"), name_it: "scegliere" },
 	{ card_id:262, name: "Corn_Dog", url: require("../pecsCards/Corn_Dog.jpg"), name_it: "Corn_Dog" },
-	{ card_id:263, name: "Stomach_ache", url: require("../pecsCards/Stomach_ache.jpg"), name_it: "Mal di stomaco" },
+	{ card_id:263, name: "Stomachache", url: require("../pecsCards/Stomachache.jpg"), name_it: "Mal di stomaco" },
 	{ card_id:264, name: "Sneeze", url: require("../pecsCards/Sneeze.jpg"), name_it: "Starnuto" },
 	{ card_id:265, name: "Water_Fountain", url: require("../pecsCards/Water_Fountain.jpg"), name_it: "Fontana d'acqua" },
-	{ card_id:266, name: "Play_at_the_beach", url: require("../pecsCards/Play_at_the_beach.jpg"), name_it: "Play_at_the_beach" },
-	{ card_id:267, name: "Ties_shoes", url: require("../pecsCards/Ties_shoes.jpg"), name_it: "Ties_shoes" },
+	{ card_id:266, name: "Play_at_beach", url: require("../pecsCards/Play_at_beach.jpg"), name_it: "Play_at_beach" },
```
Comment (Zurc) [inline src/assets/pecsCards/index.js]: Pondría 'Play_on_the_beach' ( en la arena )
https://thegrammarexchange.infopop.cc/topic/on-in-the-beach
https://www.englishforums.com/English/AtOnInTheBeach/vjbxn/post.htm

File: src/assets/pecsCards/index.js [hunk 4]
```diff
@@ -316,7 +316,7 @@ export const Data = [
 	{ card_id:315, name: "Grapes", url: require("../pecsCards/Grapes.jpg"), name_it: "Uva" },
 	{ card_id:316, name: "Thinking", url: require("../pecsCards/Thinking.jpg"), name_it: "Pensiero" },
 	{ card_id:317, name: "Lower_X", url: require("../pecsCards/Lower_X.jpg"), name_it: "Lower_X" },
-	{ card_id:318, name: "Jump_on_Trampoline", url: require("../pecsCards/Jump_on_Trampoline.jpg"), name_it: "Jump_on_Trampoline" },
+	{ card_id:318, name: "Jumpling", url: require("../pecsCards/Jumpling.jpg"), name_it: "Jumpling" },
```
Comment (Zurc) [inline src/assets/pecsCards/index.js]: 'jumpling' nunca escuché... el buscador no quiere aceptarlo... salvo como nombre de un parque de trampolines...

File: index.js [hunk 5]
```diff
@@ -3,7 +3,7 @@
  */
 
 import { AppRegistry } from 'react-native';
-import App from './App';
+import App from './app';
```
Comment (Zurc) [inline index.js:6]: están siguiendo alguna convención tipo https://github.com/airbnb/javascript ?
en los ejemplos de la página de [reactnative.dev](https://reactnative.dev/docs/assets/native-modules-ios-open-project.png) usan mayúsculas...

File: src/helpers/validators/validators.js (hunk 1 above)
Comment (jgtoriginal) [reply src/helpers/validators/validators.js]: I agree, we can do better here.
But I think this is probably good enough for now.

File: src/assets/pecsCards/index.js (hunk 2 above)
Comment (jgtoriginal) [reply src/assets/pecsCards/index.js]: Well what I'm basically doing on this regard is to follow the pattern established by the source of our cards, which I did mainly via automation. So I really don't mind naming it in a way or another, I only care of having all names the same across the board avoiding errors.

File: src/assets/pecsCards/index.js (hunk 3 above)
Comment (jgtoriginal) [reply src/assets/pecsCards/index.js]: ditto

File: src/assets/pecsCards/index.js (hunk 4 above)
Comment (jgtoriginal) [reply src/assets/pecsCards/index.js]: ditto

File: index.js (hunk 5 above)
Comment (jgtoriginal) [reply index.js:6]: we can go either way, I only fixed an error preventing us from loading the main app file on android.
convention wise, I'm thinking on migrating to ts in the near future, maybe we can have this discussion by then.

**Review by rafaelgtraverso — APPROVED** [review]

**Pull request merged**